wlauto/utils/android.py

```diff
@@ -183,6 +184,19 @@ def parse(self, apk_path):
             elif line.startswith('launchable-activity:'):
                 match = self.name_regex.search(line)
                 self.activity = match.group('name')
+            elif line.startswith('native-code'):
+                apk_abis = [entry for entry in [entry.strip() for entry in line.split(':')[1].split("'")] if entry]
```

Comment:
could be replaced with

         apk_abis = [entry.strip() for entry in line.split(':')[1].split("'") if entry.strip()]